OpenFEC-Data-Visualization/blog.md

```diff
@@ -0,0 +1,247 @@
+# Introduction to OpenFEC Data Visualization with ChartJs
+## Purpose
+
+The importance of understanding politics and the implications of being informed voters are profound. The ability for citizens of a country to understand the financial spending of candidates running for office (at any level) allows one to understand where a candidate's priorities lie. Observing and analyzing financial trends allows one to make predictions about who may win a particular election. A lot of the time, however, it’s easy to get lost in the numbers! 
+
+Thus, being able to visualize this data in a graphical manner helps in truly comparing and understanding a candidate’s financial spending in comparison to others in an easy manner. However, how do you verify the graphs you see online are indeed accurate? 
+
+It’s a catch-22 situation! Either you receive accurate numbers (hard to analyze) or potentially biased graphs. Today, you’ll learn how to make the graph yourself, using the official financial data available at the OpenFEC API.
+
+For this tutorial, we’ll walk through how to make a bar graph visualizing various financial information (operating expenses, net receipts, fundraising disbursements, and individual contribution less refunds) for Democrats and Republicans as a whole, as well as specific candidates for the 2016 presidential election.
+
+## Setup
+
+1. Sign up for the OpenFEC API Key through this [website.](https://api.data.gov/signup/)
```

Comment:
Mention specfically which website you are getting it from. What data are you getting? In addition, add a screenshot